snakefiles/mapping2.smk

```diff
@@ -4,22 +4,22 @@
 # -t: copy RG, BC and QT tags to the FASTQ header line
 rule extract_paired_reads_from_bam:
 	input: 
-		"data/input/multi_refs_positive/{run_id}/{sample}.bam"
+		"data/input/multi_refs/{run_id}/{sample}.bam"
 	output:
 		"data/input/reads/{run_id}/{sample}_paired.fastq"
 	log:
-		"logs/extract_reads_from_bam_{sample}_paired.log"
+		"logs/{run_id}/extract_reads_from_bam_{sample}_paired.log"
 	shell:
 		"samtools fastq -ntf 13 {input} > {output}"
 
 
 rule extract_singleton_reads_from_bam:
 	input: 
-		"data/input/multi_refs_positive/{run_id}/{sample}.bam"
+		"data/input/multi_refs/{run_id}/{sample}.bam"
 	output:
 		"data/input/reads/{run_id}/{sample}_singleton.fastq"
 	log:
-		"logs/extract_reads_from_bam_{sample}_singleton.log"
+		"logs/{run_id}/extract_reads_from_bam_{sample}_singleton.log"

```

Comment:
These log lines were causing an issue - all output files have to use the same set of wildcards to avoid name clashes (in this case, re-sequenced samples would overwrite the log file)